otter/integration/tests/test_lbsh.py

```diff
@@ -97,3 +98,82 @@ def test_oob_deleted_clb_node(self):
             ),
             timeout=timeout_default
         )
+
+    @tag("LBSH-002")
+    @inlineCallbacks
+    def test_move_node_to_oob_lb(self):
+        """
+        1 group, LB1 in config, LB2 not in any autoscale configs:
+
+        Server node moved from LB1 to LB2
+        Assert: Server put back on LB1
+        Assert: Server removed from LB2
+        """
+        # Create another loadbalancer not to be used in autoscale
+        # The CLB will not be added to the helper, since when the helper
+        # creates a group, it automatically adds the clb
+        clb_other = CloudLoadBalancer(pool=self.helper.pool)
+
+        yield clb_other.start(self.rcs, self)
+        yield clb_other.wait_for_state(
+            self.rcs, "ACTIVE", timeout_default)
+
+        clb_as = self.helper.clbs[0]
+
+        nodes_as = yield clb_as.list_nodes(self.rcs)
+        nodes_other = yield clb_other.list_nodes(self.rcs)
+        self.assertEqual(len(nodes_as['nodes']), 0,
+                         "There should be no nodes on the CLB yet.")
+        self.assertEqual(len(nodes_other['nodes']), 0,
+                         "There should be no nodes on the CLB yet.")
+
+        group, _ = self.helper.create_group(min_entities=1)
+        yield self.helper.start_group_and_wait(group, self.rcs)
+
+        nodes_as = yield clb_as.list_nodes(self.rcs)
+        nodes_other = yield clb_other.list_nodes(self.rcs)
+        self.assertEqual(
```

Comment:
I think this one also needs fixing in addition to the ones there (probably the initial assertion that there are zero to start with is fine).